src/css/main.less

```diff
@@ -61,6 +62,7 @@ a:focus {
 a,
 .btn {
   transition: all 0.2s ease-in-out;
+  padding: 7px 15px;
```

Comment:
Actually also about the `font-size-h2` – what does that have to do with the button?